src/main/java/com/codessquad/qna/web/exception/QnaException.java

```diff
@@ -0,0 +1,6 @@
+package com.codessquad.qna.web.exception;
+
+public class QnaException extends RuntimeException{
+
+    public QnaException(String message) { super(message); }
```

Comment:
```suggestion
    public QnaException(String message) {
        super(message);
    }
```

이것 말고도 여러개의 생성자를 선언해주면, 상속하는 Exception이 좀 더 다양한 표현을 활용할 수 있습니다.

Reply:
[RuntimeException](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/RuntimeException.html)의 다양한 생성자 확인했습니다! 다만 지금 쓰일 일이 없을 것 같아서 저것만 선언해주었습니다.

Reply:
예외 포장에 대해서도 한 번 공부해보세요~

Reply:
넵 발생하는 여러 예외들을 캐치해서 더 적절한 새로운 예외를 throw하는 방법 학습했습니다!